lesson_05/types/types_app/src/main/java/com/codedifferently/lesson5/dataprovider/AaronSantiagoProvider.java

```diff
@@ -0,0 +1,23 @@
+package com.codedifferently.lesson5.dataprovider;
+
+import java.math.BigDecimal;
+import java.util.Map;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AaronSantiagoProvider extends DataProvider {
+  public String getProviderName() {
+    return "aaronsantiago";
+  }
+
+  public Map<String, Class> getColumnTypeByName() {
+    return Map.of(
+        "column1", Long.class,
+        "column2", BigDecimal.class,
```

Comment:
In the instructions, you were directed to use the "closest possible data type." There is a different data types that would be more efficient to store the data in column2 based on its size.